tests/x86/ryujit_x86_testenv.cmd

```diff
@@ -0,0 +1,10 @@
+REM -------------------------------------------------------------------------
+REM 
+REM  This script provides x86 Ryujit test environment settings
+REM
```

Comment:
It might be useful to give an example here of how it can be used, e.g.:

runtest.cmd x86 TestEnv <full path>\ryujit_x86_testenv.cmd

I would also change REM to @REM so they don't get echo'ed to any log file.